Sua Musica/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "milrak",
+        "id": "401529002403037185"
+    },
+    "service": "Sua Música",
```

Comment:
Please change the name to `Sua Musica`, even a slight name difference can make the "Source code" button the website not work.

Reply:
or just change the folder to "Sua Música"

Reply:
> Please change the name to `Sua Musica`, even a slight name difference can make the "Source code" button the website not work.

I like change folder name, because lenguage native usign from service suamusica.com.br